assets/app/models/Github.js

```diff
@@ -173,16 +186,15 @@ var GithubModel = Backbone.Model.extend({
       return isOfType;
     });
   },
-  addPage: function (opts) {
-    opts = opts || {};
-    var content = (this.configFiles['_defaults.yml'].present) ?
-                    this.configFiles['_defaults.yml'].json : '\n',
-      commitOpts = {
-        path: opts.path,
-        message: opts.message || 'The file ' + opts.path + ' was created',
-        content: opts.content || content
-      };
-    this.commit(commitOpts);
+  getDefaults: function () {
+    var config = this.configFiles['_config.yml'],
+        defaultConfigs = (config.present && config.json) ? config.json.defaults : [],
+        defaults = defaultConfigs.filter(function (c) {
```

Comment:
@jeremiak getting an error here if `defaultConfigs` is undefined, `.filter` is not available on `undefined`. What should happen if there is not a default section of the config file?